packages/chart/src/CdcChartComponent.tsx

```diff
@@ -700,7 +631,8 @@ const CdcChart = ({
         let configCopy = { ...config }
         delete configCopy['filters']
         setConfig(configCopy)
-        setFilteredData(filterVizData(externalFilters, excludedData))
+        const filteredData = filterVizData(externalFilters, excludedData)
+        setFilteredData(filteredData)
```

Comment:
Should this use dispatch()?

Reply:
same here   const setFilteredData = filteredData => {
    dispatch({ type: 'SET_FILTERED_DATA', payload: filteredData })
  }